src/components/editor/EditorArticleController.jsx

```diff
@@ -368,18 +418,18 @@ the save changes button is supposed to be disabled in this case");
       const newAuthorsAdded = update(this.state.authorsAdded, {$splice: [[index, 1]]});
       this.safeSetState({
         authorsAdded: newAuthorsAdded,
+      }, () => {
+        console.log(this.state.authorsAdded);
       });
     }
   }
 
-  handleTeaserChanges(e) {
-    let teaser = e.target.value;
+  handleTeaserChanges() {
```

Comment:
Hmmm.... When seeing this Zane I'm starting to think it might be a bit dangerous that you refactor too much of my code. Without having tested it, I'm basically 100% sure you just broke this function completely. In React when you do a controlled component you have to manually update the value of the element, and you can only do that by getting the most recent value from the event and then updating state. I think you might've completely misunderstood what this function was for.